core/templates/domain/question/question-update.service.spec.ts

```diff
@@ -16,88 +16,38 @@
+import {StateObjectFactory} from 'domain/state/StateObjectFactory';
 
 describe('Question update service', function() {
-  var QuestionUpdateService = null;
-  var QuestionObjectFactory = null;
-  var QuestionUndoRedoService = null;
-  var StateObjectFactory = null;
-  var sampleQuestion = null;
-  var sampleStateDict = null;
-  var expectedOutputStateDict = null;
-  var expectedOutputState = null;
-  var sampleQuestionBackendObject = null;
+  let questionUpdateService = null;
+  let questionObjectFactory = null;
+  let questionUndoRedoService = null;
+  let stateObjectFactory = null;
+  let sampleQuestion = null;
+  let sampleStateDict = null;
+  let sampleQuestion1 = null;
+  let sampleStateDict1 = null;
+  let expectedOutputStateDict1 = null;
+  let expectedOutputState1 = null;
+  let expectedOutputStateDict = null;
+  let expectedOutputState = null;
+  let sampleQuestionBackendObject = null;
+  let sampleQuestionBackendObject1 = null;
   importAllAngularServices();
-
   beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    $provide.value(
-      'AnswerGroupObjectFactory', new AnswerGroupObjectFactory(
-        new OutcomeObjectFactory(),
-        new RuleObjectFactory()));
-    $provide.value(
-      'HintObjectFactory', new HintObjectFactory());
-    $provide.value(
-      'OutcomeObjectFactory', new OutcomeObjectFactory());
-    $provide.value('ParamChangeObjectFactory', new ParamChangeObjectFactory());
-    $provide.value(
-      'ParamChangesObjectFactory', new ParamChangesObjectFactory(
-        new ParamChangeObjectFactory()));
-    $provide.value('RuleObjectFactory', new RuleObjectFactory());
-    $provide.value('UnitsObjectFactory', new UnitsObjectFactory());
-    $provide.value(
-      'WrittenTranslationObjectFactory',
-      new WrittenTranslationObjectFactory());
-    $provide.value(
-      'WrittenTranslationsObjectFactory',
-      new WrittenTranslationsObjectFactory(
-        new WrittenTranslationObjectFactory()));
-  }));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
-  beforeEach(angular.mock.inject(function($injector) {
-    QuestionUpdateService = $injector.get('QuestionUpdateService');
-    QuestionObjectFactory = $injector.get('QuestionObjectFactory');
-    QuestionUndoRedoService = $injector.get('QuestionUndoRedoService');
-    StateObjectFactory = $injector.get('StateObjectFactory');
+  beforeEach(() =>{
+    questionUpdateService = TestBed.inject(QuestionUpdateService);
+    questionObjectFactory = TestBed.inject(QuestionObjectFactory);
+    questionUndoRedoService = TestBed.inject(UndoRedoService);
+    stateObjectFactory = TestBed.inject(StateObjectFactory);
 
-    sampleStateDict = {
+    sampleStateDict1 = {
```

Comment:
Avoid naming variables such that there is a trailing number that uniquely identifies it. Instead, try to give more meaningful names.

Reply:
Ditto below.

Reply:
done

Reply:
done